src/main/java/org/jabref/gui/externalfiles/UnlinkedFilesDialogViewModel.java

```diff
@@ -166,10 +188,49 @@ public void startImport() {
                                                      progressTextProperty.unbind();
                                                      taskActiveProperty.setValue(false);
                                                  })
-                                                 .onSuccess(resultList::addAll);
+                                                 .onSuccess(new Consumer<List<ImportFilesResultItemViewModel>>() {
+                                                     @Override
```

Comment:
You can use a lambda here directly no need for anonymous method, Intellij should be able to do this as well  https://www.jetbrains.com/help/rider/ConvertToLambdaExpression.html

Reply:
Replaced with a lambda now